blocks/hero/hero.css

```diff
@@ -1,31 +1,177 @@
-main .hero-container > div {
-  max-width: unset;
+/* stylelint-disable selector-class-pattern */
```

Comment:
Adding these styles temporarily. Ideally these styles should come bundled with the `udex-hero-banner` web-component. 
/cc @ok